core/templates/dev/head/footer_js_libs.html

```diff
@@ -1,5 +1,7 @@
 {% if DEV_MODE %}
   <script src="/third_party/generated/dev/js/third_party.js"></script>
+  <script src="https://npmcdn.com/angular-toastr/dist/angular-toastr.tpls.js"></script>
```

Comment:
Hi @hitesh96db  `rootDirPrefix` is concatenated with version to give the path to the compressed file to be unzipped. Here what you have done is saying that toastr will be downloaded as `angular-toastr-1.7.0.zip` which is not, It will be downloaded as realese.zip. change `url` to `https://github.com/Foxandxss/angular-toastr/archive/1.7.0.zip`  which will download it as `angular-toastr-1.7.0.zip`. Also css and js will be available inside `dist`  folder i.e `"css": ["dist/angular-toastr.min.css"],`